tests/functional/ovncontroller_controller_test.go

```diff
@@ -499,59 +495,26 @@ var _ = Describe("OVNController controller", func() {
 				daemonSetName,
 				map[string][]string{namespace + "/internalapi": {"10.0.0.1"}},
 			)
-			externalSBEndpoint := "10.0.0.254"
```

Comment:
(No change required) just making a side comment - these tests seem to target ovndbcluster controller, but they are located in ovncontroller test file. We should probably relocate them.